### ISSUE TITLE
Monitoring the Italian National and Regional Contagion Trends

### ISSUE THREAD

**Issue by ilaria-kode** [Milano-R/erum2020-covidr-contest#91]
Submit your contributions by filling the following information

``` yaml
# Contribution's title, author, abstract
title: MINT - Monitoring the Italian National and Regional Contagion Trends
author: Ilaria Ceppa et al., Kode Solutions
abstract: |
  MINT is a Shiny dashboard developed to visualize and analyze in a simplified manner data on the spread of the COVID-19 virus in Italy.
  The web app uses publicly available data, published every day by Protezione Civile Italiana on github. The app automatically updates every day at 6:30pm.
  In the first tab the dashboard provides an overview of national data, visualizing trends and percent increments of various variables, whereas in the tab "Regional data" the user can select a specific region and explore the data at a regional level.
  Moreover, in the "Regional Trends" tab, we give the user an overall view of the trends of specific variables across all the regions, while in the "Regional Comparison" tab the user can compare two or more specific regions in terms of their trends and percent increment values over time.
  Finally, in the tab "Twitter overview", the app shows the trends for the number of sent tweets and hashtags searches. This data was collected from Twitter from the end of February to the beginning of April. 
# Short name to be used in the website Gallery menu (can be the same as title)
menu_entry: MINT - Monitoring Italian contagion trends
# URL of the public source code repository
repository: https://gitlab.com/kode_public/app-pc-data
# For more complex / heterogeneous contributions, you can create a GitHub
# Gist (https://gist.github.com/) with the relevant information and pointers:
type: gist
content: <script src="https://gist.github.com/ilaria-kode/a37ba1b4a661f9208a8dbed3bfab5fa0.js"></script>
```

In addition:

- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: Issue #00", where #00 is the number of this Issue
     - Content: A full link to this Issue
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.
     
If you have questions specific to your submission, used the comments in this Issue.

**Comment by riccardoporreca** [comment]
@ilaria-kode, thansks for your submission, I will just shorten the `author` and `menu_entry` as follows for a better look in the gallery
```yml
author: Ilaria Ceppa et al., Kode Solutions
menu_entry: MINT - Monitoring Italian contagion trends
```

**Comment by ilaria-kode** [comment]
That's fine for us. I updated the issue. Thank you!

**Comment by riccardoporreca** [comment]
There are currently issues with GitHub preventing the integration of contributions (https://www.githubstatus.com/).
We will finalize submissions once such issues are solved.

**Comment by riccardoporreca** [comment]
Merged corresponding PR #96.
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
@ilaria-kode, the major GitHub incident was solved and now contributions are fully integrated.

Your contribution is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/kode-mint.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686
